Xamarin.Forms.ControlGallery.Android/FormsAppCompatActivity.cs

```diff
@@ -42,13 +42,12 @@ protected override void OnCreate(Bundle bundle)
 
 #if TEST_EXPERIMENTAL_RENDERERS
 			// CollectionView lets us test CollectionView stuff until it's officially released
-			Forms.SetFlags("FastRenderers_Experimental");
+			Forms.SetFlags("FastRenderers_Experimental"/*, "CollectionView_Experimental", "Visual_Experimental"*/); 
```

Comment:
let's add the flag on always no ? So we can write UITests for it ?

Reply:
The main reason we left these out for these PRs was due to wanting to verify proper behavior with the flags disabled since these are experimental.  This way we verify if someone installs a new package and doesn't set these it doesn't blow up for them. 

The next evolution we can probably just leave the flags on